bootstrap.sh

```diff
@@ -50,7 +50,8 @@ find_variant()
   done
   if [ "x$found" = "xNOT_FOUND" ]; then
     echo "WARNING: Cannot find $tool version $versions" >&2
-    echo "Trying `$tool --version | head -1`" >&2
+    echo -n "Trying " >&2
+    ($tool --version | head -1) >&2
```

Comment:
If we go to the model that stdout can be ignored, while stderr should report error conditions, then these should go to stdout, shouldn't they?

Also, please be aware that not all OSes support "echo -n" (MacOS doesn't, for instance)

Reply:
> This is the report of what went wrong when calling the tools `--version` parameter.
> 
> Thanks for the reminder about `-n` on MacOS. I guess we can do without that line entirely.

A pattern that could work on macos is:
toolversion=`$tool --version | head -1`
```suggestion
echo "Trying $toolversion" >&2 
```